toonz/sources/toonzfarm/tfarmserver/tfarmserver.cpp

```diff
@@ -702,6 +708,14 @@ void FarmServer::queryHwInfo(HwInfo &hwInfo)
 	len = sizeof(physMemSize);
 	sysctl(mib, 2, &physMemSize, &len, NULL, 0);
 
+#elif defined(LINUX)
+	physMemSize = (TINT64)sysconf(_SC_PHYS_PAGES) * (TINT64)sysconf(_SC_PAGE_SIZE);
```

Comment:
No, if you read the #if/#else#endif flow carefully you will see it is only set properly on win32, both OSX, Linux and other platforms fallback to the 500 MB for available physical memory and total/available virtual memory.